test/views/responsiblepeople/your_answersSpec.scala

```diff
@@ -30,21 +33,81 @@ class your_answersSpec extends AmlsSpec with MustMatchers {
 
+      html must include(Messages("responsiblepeople.whomustregister.line_1"))
+      html must include(Messages("responsiblepeople.whomustregister.line_2"))
+      html must include(Messages("responsiblepeople.whomustregister.line_3"))
+      html must include(Messages("responsiblepeople.whomustregister.line_4"))
+      html must include(Messages("responsiblepeople.whomustregister.line_5"))
+
+    }
+
+    "have list with Incomplete and Complete headers displayed when there are both types of lists" in new ViewFixture {
+      def  view = views.html.responsiblepeople.your_answers(completeRpSeq, incompleteRpSeq)
+
+      html must include(Messages("responsiblepeople.check_your_answers.incomplete"))
+      html must include(Messages("responsiblepeople.check_your_answers.complete"))
+
+    }
+
+    "have list with Incomplete header displayed when there are only incomplete RP's" in new ViewFixture {
+      def  view = views.html.responsiblepeople.your_answers(Seq.empty[(ResponsiblePerson, Int)], incompleteRpSeq)
+
+      html must include(Messages("responsiblepeople.check_your_answers.incomplete"))
+      html must not include Messages("responsiblepeople.check_your_answers.complete")
+
+    }
+
+    "have list without Complete/Incomplete headers displayed when there are only complete RP's" in new ViewFixture {
+      def  view = views.html.responsiblepeople.your_answers(completeRpSeq, Seq.empty[(ResponsiblePerson, Int)])
+
+      html must not include Messages("responsiblepeople.check_your_answers.incomplete")
+      html must not include Messages("responsiblepeople.check_your_answers.complete")
+    }
+
+    "have an add a responsible person link with the correct text and going to the what you need page" in new ViewFixture {
+      def  view = views.html.responsiblepeople.your_answers(incompleteRpSeq, completeRpSeq)
+
+      doc.getElementById("addResponsiblePerson").text must be(Messages("responsiblepeople.check_your_answers.add"))
+      doc.getElementById("addResponsiblePerson").attr("href") must be(controllers.responsiblepeople.routes.ResponsiblePeopleAddController.get(false).url)
+    }
+
+    "have an incomplete/complete sections with people names displayed and edit/remove links" in new ViewFixture {
+      def view = views.html.responsiblepeople.your_answers(incompleteRpSeq, completeRpSeq)
+
+      doc.getElementById("incomplete-header").text must include(Messages("responsiblepeople.check_your_answers.incomplete"))
+      html must include("Katie Test")
+      doc.getElementById("detail-edit-1").attr("href") must be(controllers.responsiblepeople.routes.PersonNameController.get(2, false, None).url)
+      doc.getElementById("detail-remove-1").attr("href") must be(controllers.responsiblepeople.routes.RemoveResponsiblePersonController.get(2, true).url)
+      doc.getElementById("complete-header").text must include(Messages("responsiblepeople.check_your_answers.complete"))
+      html must include("John Test")
```

Comment:
Should check that this is inside the incomplete section

Reply:
done